web/forms/implementing_simple_validation/my_element.dart

```diff
@@ -1,25 +1,27 @@
 @HtmlImport('my_element.html')
```

Comment:
I think this example shows the very basics how to implement something like validation yourself. 
Using iron-, paper-, xxx-elements would probably fit better into the `polymer_elements` examples. The examples in the `web/polymer_elements` subfolder in this repo are very basic and just show how to add a toolbar.
If you prefer to use iron_validator_behavior I'll change it of course.